omod-common/src/test/java/org/mockingbird/test/rest/resource/AnimalClassResource_1_9.java

```diff
@@ -9,6 +9,7 @@
  */
```

Comment:
@dkayiwa What do you think? Should we have these tests? On the surface, `AnimalClassResource` doesn't seem right inside an OpenMRS webservices module. Ideally we should be replacing these tests?